src/sentry/integrations/utils/code_mapping.py

```diff
@@ -3,7 +3,7 @@
 
 from .repo import Repo, RepoTree
 
-logger = logging.getLogger("sentry.integrations.utils.code_mapping")
+logger = logging.getLogger(__name__)
```

Comment:
This is the same value. If we move the file we will not need to remember to update this.